docs/api/connection.md

```diff
@@ -68,7 +68,7 @@ Optional
 
 > Default: `{}`
 
-> `options.results` -- Object[]
+> `options.results` -- [`Doc`]({{ site.baseurl }}{% link api/doc.md %})[]
 
 > > Prior query results, if available, such as from server rendering
```

Comment:
```suggestion
> > Prior query results, if available, such as from server rendering. This should be an array of Doc instances, as obtained from `connection.get(collection, id)`. For best results, invoke `ingestSnapshot` for each Doc instance to avoid transferring data for each document.
```